src/components/table/table.js

```diff
@@ -72,8 +72,13 @@ const CustomTable = ({ mobileCaption, columns, rows, enableSelection = true, act
                 </TableCell>
               )}
               {columns.map((column) => (
+
                 <TableCell key={`${row.id}-${column.id}`} width={column.width} align={column.align || 'left'}>
-                  {column.display(row)}
+                  {typeof row[column.field]== 'string' && row[column.field].includes('\n')? 
+                  row[column.field].split('\n').map((item, i) => <p key={i}>{item}</p>)
+                  : column.display(row)//I need to replace this by row[column.field] to make dashboard works
```

Comment:
@KariGarcia this is the part of the conflict, I was working with the row[column.field] version while you changed it to column.display(row). It doesn't work for the dashboard as we have 2 text fields concatenated in one single cell.